tests/logger.js

```diff
@@ -76,6 +76,13 @@ exports.set_format = {
         test.equal(this.logger.format, this.logger.formats.DEFAULT);
         test.done();
     },
+    'set format to EMERG' : function (test) {
+        test.expect(1);
+        this.logger.format = '';
+        this.logger.set_format('EMERG');
+        test.equal(this.logger.format, this.logger.formats.EMERG);
```

Comment:
I think your problem is here. You want to compare against logger.levels.EMERG.

Reply:
d'oh

Reply:
While you're in there, maybe also add a test for `this.logger.set_format('emerg');` as well, since there doesn't appear to be any lower case tests.

Reply:
you mean set_level?

Reply:
LOL, yes. That's what I meant. :)

Reply:
@KingNoosh, you going to fix this or what? My merge finger is getting itchy.